docs/validator/requirements.md

```diff
@@ -55,10 +55,10 @@ An attacker scans the network, tries to learn the IP address of various validato
 
 One recommended way to mitigate these risks is for validators to carefully structure their network topology in a so-called sentry node architecture.
 
-Validator nodes should only connect to full-nodes they trust because they operate them themselves or are run by other validators they know socially. A validator node will typically run in a data center. Most data centers provide direct links the networks of major cloud providers. The validator can use those links to connect to sentry nodes in the cloud. This shifts the burden of denial-of-service from the validator's node directly to its sentry nodes, and may require new sentry nodes be spun up or activated to mitigate attacks on existing ones.
+Validator nodes are expected to connect only to full-nodes they trust because they operate them themselves or are run by other validators they know socially. A validator node typically runs in a data center. Most data centers provide direct links the networks of major cloud providers. The validator can use those links to connect to sentry nodes in the cloud. This shifts the burden of denial-of-service from the validator's node directly to its sentry nodes, and may require new sentry nodes be spun up or activated to mitigate attacks on existing ones.
```

Comment:
should is ambiguous and I'm on a mission to eliminate ambiguity!
I rely on the [Google developer documentation style guide](https://developers.google.com/style/word-list#letter-s) 
<img width="969" alt="image" src="https://user-images.githubusercontent.com/10067215/169328619-969e70a7-1c2b-4e70-8c77-6972f72616b4.png">

Reply:
thanks Barrie